kong/runloop/balancer/targets.lua

```diff
@@ -511,15 +512,21 @@ local function targetExpired(target)
   return not target.lastQuery or target.lastQuery.expire < ngx_now()
 end
 
+local function get_updated_now_ms()
+  update_time()
+  return ngx_now() * 1000 -- time is kept in seconds with millisecond resolution.
+end
```

Comment:
Just a small suggestion, this function appears twice, could it be placed in a module like tools or utils.